lua/lazyvim/plugins/extras/lang/php.lua

```diff
@@ -48,7 +48,7 @@ return {
     optional = true,
     opts = function()
       local dap = require("dap")
-      local path = require("mason-registry").get_package("php-debug-adapter"):get_install_path()
+      local path = LazyVim.get_pkg_path("php-debug-adapter")
```

Comment:
This can just be:

```lua
dap.adapters.php = {
  type = "executable",
  command = "php-debug-adapter",
  args = {}
}
```

Reply:
Nice catch. `php-debug-adapter`'s [package manifest](https://github.com/mason-org/mason-registry/blob/0d12009f2cc502317f7a3cc2b5f1875e30dd5fd6/packages/php-debug-adapter/package.yaml#L18) does export this as a bin, so the command can just be that.

However, this mason dependency doesn't seem to be installed anywhere. Shouldn't we add a spec to this `nvim-dap` spec to add an `ensure_installed` for it?

Reply:
Btw, just noticed you're the Mason creator. Thanks for your work!